drivers/volume/portworx/portworx.go

```diff
@@ -419,6 +419,14 @@ func (d *portworx) Init(sched, nodeDriver, token, storageProvisioner, csiGeneric
 }
 
 func (d *portworx) RefreshDriverEndpoints() error {
+
+	// namespace of portworx in switched context mmight've changed
```

Comment:
nit: typo in `might`

Reply:
Perhaps it would be helpful to print the namespace if the namespace of the portworx is changed after switching context, as this behavior may not always be expected.

Reply:
✅